### PR TITLE
Prototype of the the CTF skimming workflow

### PR DESCRIPTION
**Pull request by shahor02**
Format: ctf-skim-workflow.sh -f <IRFramesFile> -c <CTFsList>
where IRFramesFile is the output of IRFrame selection workflow (tree or vector format) CTFsList: CTF (as a textfile or comma-separated list) fully covered by IRFramesFile

For the record: with the test IRFrame selection provided by @mpuccio the skimmed CTF yield is 10% of the original.

### CODE REVIEW

File: DATA/production/ctf-skim-workflow.sh [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+{ source $O2DPG_ROOT/DATA/production/workflow-multiplicities.sh; [[ $? != 0 ]] && echo "workflow-multiplicities.sh failed" 1>&2 && exit 1; }
+
+WORKFLOW= # Make sure we start with an empty workflow
+
+POSITIONAL=()
+while [[ $# -gt 0 ]]; do
+  key="$1"
+  case $key in
+    -f|--irframes-selection)
+      IRFREAMES="$2"
```
Comment (chiarazampolli) [inline DATA/production/ctf-skim-workflow.sh:32]: Here you have a typo, consistent throughout the script:
IRFREAMES --> IRFRAMES

Comment (shahor02) [reply DATA/production/ctf-skim-workflow.sh]: Thanks for spotting, fixed.

**Review by chiarazampolli — CHANGES_REQUESTED** [review]
Before merging, can you put this in a dedicated folder? E.g.:

O2DPG/DATA/production/configuration/CTFSkimming

You should then check that all sourcing work fine.

Thanks!

**Comment by shahor02** [comment]
Depends on https://github.com/AliceO2Group/AliceO2/pull/10855

**Pull request merged**